Corona-Warn-App/src/main/res/layout/fragment_greencertificate_details.xml

```diff
@@ -0,0 +1,334 @@
+                        android:layout_height="wrap_content"
+                        android:layout_marginEnd="72dp"
+                        android:importantForAccessibility="no"
+                        app:srcCompat="@drawable/ic_cwa_logo_white" />
+                </LinearLayout>
+
+            </com.google.android.material.appbar.MaterialToolbar>
+
+        </com.google.android.material.appbar.CollapsingToolbarLayout>
+
+    </com.google.android.material.appbar.AppBarLayout>
+
+    <androidx.core.widget.NestedScrollView
+        android:id="@+id/scroll_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:layout_behavior="@string/appbar_scrolling_view_behavior">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginVertical="16dp"
+            android:orientation="vertical">
+
+            <include
+                android:id="@+id/qr_code_card"
+                layout="@layout/greencertificate_qrcode_card"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginHorizontal="24dp"
+                android:layout_marginBottom="6dp" />
+
+            <LinearLayout
+                style="@style/Card.Greencertificate"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginHorizontal="24dp"
+                android:orientation="vertical"
+                android:padding="16dp">
+
+                <TextView
+                    style="@style/body1Medium"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="12dp"
+                    android:text="@string/green_certificate_attribute_name" />
+
+                <TextView
+                    android:id="@+id/name"
+                    style="@style/body1"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:text="Andrea Schneider"
+                    tools:text="Andrea Schneider" />
+
+                <TextView
+                    style="@style/body1Medium"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="24dp"
+                    android:text="@string/green_certificate_attribute_dob" />
+
+                <TextView
+                    android:id="@+id/birth_date"
+                    style="@style/body1"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="4dp"
+                    android:text="geboren 18.04.1943"
```

Comment:
there is no `geboren` in figma it is just the date

Reply:
This is currently only a placeholder to test the layout in the test_nav_graph